### DISCUSSION TITLE
Auto filling properties from migration file

### DISCUSSION THREAD

**Discussion by johnngondi** [spatie/laravel-data#780] [Q&A]
Hello,

There was a time I created laravel data using the command and it automatically prefilled properties using the model migration.
I don't know what I did to make it do that but I have never seen it again, how can I redo it?
Or was I high?

**Comment by rubenvanassche** [comment]
I'm afraid it was 4:20 😅

**Reply by johnngondi** [reply]
I am quiting that thing. It is a nice feature though.

**Reply by rubenvanassche** [reply]
It never existed 😛
